### ISSUE TITLE
Should it work via microusb?

### ISSUE THREAD

**Issue by Ameb** [ALSchwalm/pISO#97]
I had multiple problems with the pogo pins and now i cant remember if the computer should recognize drives when the piso is connected via pi zero microusb ports. Because mine does not. I had to repair multiple pogo pins.

Thank you.

**Comment by ArdjanB** [comment]
Yes, it connects to the PC the same, whether you use the ‚main‘ USB port or the one of the two micro USB ports.
